### PR TITLE
Updates for auto modes

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates docs for auto modes.  Primarily changes add links to the more detailed docs. I have also removed some detail that isn't needed.

### CODE REVIEW

File: en/getting_started/flight_modes.md [hunk 1]
```diff
@@ -111,13 +111,14 @@ For multicopters this is the 3D flying mode. You can make flips and acrobatics.
 
 [<img src="../../assets/site/automatic_mode.svg" title="Automatic mode" width="30px" />](#key_automatic)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](#key_position_fixed)
 
-You can use Hold in case of emergency or during a mission in order to Hold the aircraft at its current position autonomously and pause the flight, while you get time to assess the situation either emergency or mission related. When you activate Hold mode with a pre-programmed switch, the multicopter will fix itself to its current GPS position and altitude and hover there. During hold you are not able to control the craft with the RC. It is the first line of defense if you lose control over the aircraft and panic.
```
Comment (hamishwillee) [inline en/getting_started/flight_modes.md]: FYI removed line "It is the first line of defense if you lose control over the aircraft and panic."
1. that depends on your current mode - you might just centre your sticks
2. Captured in the text "help regain control of vehicle"

File: en/getting_started/flight_modes.md [hunk 2]
```diff
@@ -111,13 +111,14 @@ For multicopters this is the 3D flying mode. You can make flips and acrobatics.
...
+[Hold mode](../flight_modes/hold.md) causes the multicopter to stop and hover at its current position and altitude (maintaining position against wind and other forces). The mode can be used to pause a mission or to help regain control of a vehicle in an emergency. It is usually activated with a pre-programmed RC switch.
+
 
 ### Return Mode {#return_mc}
 
 [<img src="../../assets/site/automatic_mode.svg" title="Automatic mode" width="30px" />](#key_automatic)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](#key_position_fixed)
 
-The multicopter will return in a straight line at the current altitude (if higher than the home position + hovering altitude) or on the hovering altitude (if higher than the current altitude), then lands automatically.
+[Return mode](../flight_modes/return.md) causes the vehicle to return (at a safe height) to its home position and land. The mode may be activated manually (via a pre-programmed RC switch) or automatically (i.e. in the event of a failsafe being triggered).
```
Comment (hamishwillee) [inline en/getting_started/flight_modes.md]: At this level people don't need to know about various settings - they just need to know it is travelling back at safe height. They also don't need to know that this is configurable - that is for the linked doc. Here I'm just trying to capture the default behaviour and "flavour" of behaviour.

File: en/getting_started/flight_modes.md [hunk 3]
```diff
@@ -133,24 +134,29 @@ The multicopter will return in a straight line at the current altitude (if highe
...
+[Land mode](../flight_modes/land.md) causes the multicopter to land at the location at which the mode was engaged.
 
 
 ### Follow Me Mode {#followme_mc}
 
 [<img src="../../assets/site/automatic_mode.svg" title="Automatic mode" width="30px" />](#key_automatic)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](#key_position_fixed)
 
+[Follow Me mode](../flight_modes/follow_me.md) causes a multicopter to autonomously follow and track a user providing their current position setpoint. Position setpoints might come from an Android phone/tablet running *QGroundControl* or from a Dronecode SDK app.
 
+### Offboard Mode {#offboard_mc}
```
Comment (hamishwillee) [inline en/getting_started/flight_modes.md]: Added this mode. Most users won't need to know about it, but I figure we provide information that this is something they can mostly ignore.

File: en/getting_started/flight_modes.md (hunk 1 above)
Comment (bkueng) [reply en/getting_started/flight_modes.md]: Yep that is good. You can also activate it from QGC (Pause button)

**Comment by hamishwillee** [comment]
Thanks @bkueng !

**Pull request merged**